crates/sui-framework/packages/sui-framework/sources/tx_context.move

```diff
@@ -139,3 +143,9 @@ public fun increment_epoch_number(self: &mut TxContext) {
 public fun increment_epoch_timestamp(self: &mut TxContext, delta_ms: u64) {
     self.epoch_timestamp_ms = self.epoch_timestamp_ms + delta_ms
 }
+
+#[test_only]
+public fun set_epoch(self: &mut TxContext, epoch: u64) {
```

Comment:
Should we call this `advance_epoch`? since it can't go backwards

Reply:
We have APIs that should help address this already in `test_scenario`. I would vote we continue to use those and not build out competing APIs